source/common/grpc/google_async_client_impl.cc

```diff
@@ -468,6 +468,9 @@ void GoogleAsyncRequestImpl::onCreateInitialMetadata(Http::RequestHeaderMap& met
   callbacks_.onCreateInitialMetadata(metadata);
 }
 
+// Only relevant to determine whether the xDS stream is reachable.
```

Comment:
Why isn't this relevant if Google gRPC is used for xDS?

Reply:
Apologies for not making it clear.
This PR only introduces `onServiceRechable()` and implements it for EnvoyGrpc (trying to keep things small).
The GoogleGrpc implementation will be part of a future PR.
The comment is to say that eventually it will only be used for xDS-streams rather than to any GoogleGrpc stream (e.g., ext-proc). In the future, this may be changed, but I don't see that need at the moment.

Reply:
Fair - do you have link to a PR showing how this works for GoogleGrpc? Just curious.